static/app/components/replays/videoReplayer.tsx

```diff
@@ -101,6 +109,22 @@ export class VideoReplayer {
         this._callbacks.onLoaded(event);
       }
     });
+
+    // Finished loading data, ready to play
+    el.addEventListener('loadeddata', () => {
```

Comment:
Should we unsubscribe these events when/if the `el` is removed from the dom? I wonder if we're leaking memory right now because there are so many video elements cycling in and out.

Reply:
Yeah, there's lots of cleanups that need to happen, `_videos` and `_timer` as well I think